package.json

```diff
@@ -50,6 +50,7 @@
     "countries-and-timezones": "^3.6.0",
     "dotenv": "^16.4.4",
     "ethers": "^5.7.2",
+    "@ariesgun/ethers-decode-error": "^1.1.0",
```

Comment:
I think we should self host @rndquu

Reply:
Is it something I can do?

Reply:
Perhaps we need to get a copy of your repo and then host it. @rndquu @whilefoo @gentlementlegen

Reply:
Yeah we do.

Also Ana shared QA on another issue that showed the toaster displaying the same error that aries provided as QA, so there is little difference compared to what prod is running rn as far as I can tell.

We should use 4byte and pull the onchain fn names into the console using the selector from the data in the call that fails. Saves having to manually do it while debugging.